jakarta-xml-binding/Dockerfile

```diff
@@ -1,8 +1,8 @@
 # Reference: https://docs.docker.com/engine/reference/builder/
 
-FROM fedora:35
+FROM fedora:39
 RUN ["dnf", "upgrade", "--refresh", "-y"]
-RUN ["dnf", "install", "-y", "bash-completion", "ant", "java-11-openjdk", "java-11-openjdk-devel", "java-11-openjdk-jmods", "java-17-openjdk", "java-17-openjdk-devel", "java-17-openjdk-jmods", "java-latest-openjdk", "java-latest-openjdk-devel", "java-latest-openjdk-jmods", "maven", "maven-openjdk11", "git", "git-extras", "wget", "unzip", "procps-ng", "file"]
```

Comment:
I think we should add Java 21 JDK as is done by this pull request but also keep adding Java 11/17.

Reply:
+1 we should keep both 11 and 17.

Reply:
So we don't create a separate branch for running tck with java21 ?  After "-DnoSecurityManager=true" is added, it is all running without security manager.